<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElevatorP</class>
 <widget class="QWidget" name="ElevatorP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>260</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="floor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;|&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;|&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 97, 81);</string>
   </property>
   <property name="text">
    <string>HELP</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Obstacle</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="fireSignal">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Fire Signal</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Power Outage</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="pResponse">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Passenger respond?</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="safety">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Building safety respond?</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Overload</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
